actions.go

```diff
@@ -5,11 +5,13 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	"regexp"
+	"strings"
 )
 
 // PerformScan runs a scan of a given host using a script with scriptName.
 // At this moment, we assume that only MAC addresses will be created/updated/deleted in Ralph.
-func PerformScan(addrStr, scriptName string, dryRun bool, cfg *Config, cfgDir string) {
+func PerformScan(addrStr, scriptName string, withModel, dryRun bool, cfg *Config, cfgDir string) {
```

Comment:
You can group args of the same type in Go, e.g. instead of writing `withModel bool, dryRun bool` you can write `withModel, dryRun bool` - it is especially useful with functions with long signatures.